option optimize_for = LITE_RUNTIME;

message HeroUpgradeDestinyAnswer
{
 required uint32 result = 1;
 required uint32 exp = 2;
}
